mobile/src/app/components/create-protest/create-protest.component.html

```diff
@@ -0,0 +1,3 @@
+<p>
```

Comment:
Might be worth it to make `protests` the containing component and move all the content from the `ProtestsComponent` in to the `CreateProtestComponent.`